src/h3lib/include/h3api.h.in

```diff
@@ -262,6 +262,17 @@ double H3_EXPORT(hexAreaKm2)(int res);
 double H3_EXPORT(hexAreaM2)(int res);
 /** @} */
 
+/** @brief "great circle distance" between pairs of GeoCoord points */
+// todo: do we want pass-by-reference or by value?
```

Comment:
Not my area of specialty, but I believe we generally pass structs we don't intend to modify by `const` pointers (ie reference). I assume that this is more efficient from a memory perspective.